sockeye/extract.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
`extract` is a rather generic name, how about `extract_parameters` or `convert_parameters` ?
It would also make sense to add this CLI as an entry point to setup.py, something like `sockeye-extract-parameters`.